tests/test_website.py

```
@@ -1,25 +1,41 @@
 import unittest
-from unittest.mock import patch, MagicMock 
+from unittest.mock import patch, MagicMock, mock_open, call
```

Ruff (0.11.9)

`unittest.mock.call` imported but unused

Remove unused import: `unittest.mock.call`

(F401)